OpenSourcePolitics/decidim-app
# feat: Addition of the module emitter

#### :tophat: Description

This PR adds the module Emitter with addition of related specs

#### :pushpin: Related Issues
- [Ajout module Emitter Decidim-app](https://opensourcepolitics.odoo.com/web?db=opensourcepolitics&signup_email=guillaume%40opensourcepolitics.eu&token=ixqNZ9S5lGRCj9ADpsQq#id=3406&cids=1&menu_id=325&action=471&model=project.task&view_type=form)

#### Testing

Example:
* Log in as an admin
* Access to the Backoffice
* Go to a participatory process
* Edit it
* Scroll down until you see the "Emitter" part
* Create a new emitter with a name and an image
* Save it
* Access front office and make sure the emitter is appearing in the hompeage or Participatory Processes index
* Go back the the BO
* Edit another Participatory Process
* Try to see in the emitter part if the emitter you previously created appears in the scrolldown
* Try to edit the PP


#### Tasks
- [x] Addition of the module
- [x] Add the new migration 
- [x] Override of the admin form of participatory processes
- [x] Add new specs related to the module emitter
